### PR TITLE
[3006.x] Include the version of `relenv` in the versions report.

### PR DESCRIPTION
**Pull request by s0undt3ch**
### What does this PR do?
See title.

### What issues does this PR fix or reference?
Fixes: #63827

### CODE REVIEW

File: tests/pytests/functional/cli/test_salt.py [hunk 1]
```diff
@@ -17,6 +17,7 @@ def test_versions_report(salt_cli):
                 section[key] = section[key].strip()
 
     ret = salt_cli.run("--versions-report")
+    assert ret.returncode == 0
```
Comment (Ch3LL) [inline tests/pytests/functional/cli/test_salt.py:20]: Can we test "relenv" exists in the output of `--versions-report` if we are testing a onedir build?

**Review by Ch3LL — CHANGES_REQUESTED** [review]

File: tests/pytests/functional/cli/test_salt.py (hunk 1 above)
Comment (s0undt3ch) [reply tests/pytests/functional/cli/test_salt.py:20]: Oh, specifically of we're on a Onedir right?

Comment (s0undt3ch) [reply tests/pytests/functional/cli/test_salt.py:22]: Pushed a test update.

**Review by Ch3LL — APPROVED** [review]

**Review by whytewolf — APPROVED** [review]

**Comment by s0undt3ch** [comment]
Approved and passing. Merging.

**Pull request merged**